### PR TITLE
Giocomo sbx imaging datainterfaces

### PR DESCRIPTION
**Pull request by Saksham20**
adds 
1) Interface for the Giocomo labs Plitt VR data.
2) Creates a NWBConverter child class for the complete conversion

### CODE REVIEW

**Comment by Saksham20** [comment]
will pass when #132 merges

File: nwb_conversion_tools/datainterfaces/giocomoconverter.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from ..nwbconverter import NWBConverter
+from .imagingextractorinterface import SbxImagingInterface
+from .roiextractordatainterface import Suite2pSegmentationInterface
+from .giocomovrdatainterface import GiocomoVRInterface
```
Comment (bendichter) [inline nwb_conversion_tools/datainterfaces/giocomoconverter.py:4]: The Giocomo converter should go in [giocomo-lab-to-nwb](https://github.com/catalystneuro/giocomo-lab-to-nwb)

File: nwb_conversion_tools/datainterfaces/giocomoconverter.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from pathlib import Path
+import warnings
+
+
+class GiocomoImagingInterface(NWBConverter):
```
Comment (bendichter) [inline nwb_conversion_tools/datainterfaces/giocomoconverter.py:9]: Anything "Giocomo" should go in https://github.com/catalystneuro/giocomo-lab-to-nwb

**Review by bendichter — CHANGES_REQUESTED** [review]
these changes are in the wrong repo

**Comment by bendichter** [comment]
@Saksham20 please make sure that all lab-specific converters and interfaces are added to the lab repo, not NWB Conversion Tools. SBXDataInterface goes in this repo, but GiocomoNWBConverter and all Giocomo-specific interfaces do not

**Pull request closed without merging**